client/src/components/Amplitude.ts

```diff
@@ -27,7 +27,44 @@ type AmplitudeEvent =
   | "hpdRegistrationIsIncomplete"
   | "hpdRegistrationNotRequired"
   | "hpdRegistrationMaybeRequired"
-  | "hpdRegistrationRequiredAndNotThere";
+  | "hpdRegistrationRequiredAndNotThere"
+  | "navbarHowToUse"
+  | "switchToEnglish"
+  | "switchToSpanish"
+  | "emailSignUp"
+  | "newSearch"
+  | "acris-overview-tab"
+  | "hpd-overview-tab"
+  | "dob-overview-tab"
+  | "dof-overview-tab"
+  | "dap-overview-tab"
+  | "acris-timeline-tab"
+  | "hpd-timeline-tab"
+  | "dob-timeline-tab"
+  | "dof-timeline-tab"
+  | "dap-timeline-tab"
+  | "acris-not-registered-page"
+  | "hpd-not-registered-page"
+  | "dob-not-registered-page"
+  | "dof-not-registered-page"
+  | "dap-not-registered-page"
+  | "numAddrsClick"
+  | "timelineTab"
+  | "portfolioTab"
+  | "summaryTab"
+  | "whoIsLandlordAccordian"
+  | "detailsOpenContactCard"
```

Comment:
Hmmm so some of these are in kebab-case and some are in camelCase. Maybe we should consider defaulting just to camelCase since that's the case that previous event labels have used? Not a huge dealbreaker tho.

Reply:
Yeah, those non camelCase ones are because of the template strings we need to use in some places, and not wanting to complicate the code in each log step by doing a bunch of string cleaning (especially since the google ones use kebab-case (love that name btw!)